packages/tooltip/test/tooltip.test.ts

```diff
@@ -214,6 +214,19 @@ describe('Tooltip', () => {
 
         expect(typeof el.tipElement).to.not.equal('undefined');
     });
+
+    it('self managed does not attach event listeners if no trigger was found', async () => {
+        const documentEventsSpy = spy(document, 'addEventListener');
+        const el = await fixture<HTMLDivElement>(
+            html`
+                <sp-tooltip self-managed>Help text.</sp-tooltip>
+            `
+        );
+
+        await elementUpdated(el);
+        expect(documentEventsSpy.callCount).to.equal(0);
```

Comment:
Can we confirm it doesn't attach to anything else, too?
```suggestion
        expect(documentEventsSpy.callCount).to.equal(0);
        expect(el.overlayElement?.triggerElement).to.be.null;
```